.github/meta/update_labels_and_milestones.py

```diff
@@ -0,0 +1,196 @@
+from github import Github
+
+
+repos = [
+    "OpenSlides",
+    "openslides-auth-service",
+    "openslides-autoupdate-service",
+    "openslides-backend",
+    "openslides-client",
+    "openslides-datastore-service",
+    "openslides-icc-service",
+    "openslides-manage-service",
+    "openslides-media-service",
+    "openslides-vote-service",
+    "vote-decrypt",
+]
+
+labels = {
+    "general": [
+        {
+            "name": "blocker",
+            "color": "fbca04",
+        },
+        {
+            "name": "bug",
+            "color": "b60205",
+        },
+        {
+            "name": "clean up",
+            "color": "c5def5",
+        },
+        {
+            "name": "critical",
+            "color": "000000",
+            "description": "Highest priority: This has to be done first."
+        },
+        {
+            "name": "dependencies",
+            "color": "c5def5",
+        },
+        {
+            "name": "documentation",
+            "color": "c5def5",
+        },
+        {
+            "name": "enhancement",
+            "color": "006b75",
+            "description": "General enhancement which is neither bug nor feature"
+        },
+        {
+            "name": "feature",
+            "color": "0e8a16",
+        },
+        {
+            "name": "good first issue",
+            "color": "5319e7",
+        },
+        {
+            "name": "high",
+            "color": "d93f0b",
+        },
+        {
+            "name": "low",
+            "color": "fbca04",
+        },
+        {
+            "name": "needs discussion",
+            "color": "c5def5",
+        },
+        {
+            "name": "needs info",
+            "color": "c5def5",
+        },
+        {
+            "name": "needs investigation",
+            "color": "e99695",
+        },
+        {
+            "name": "performance",
+            "color": "006b75",
+        },
+        {
+            "name": "waiting",
+            "color": "cccccc",
+            "description": "Waiting for some other PR/feature; more details in comments"
+        },
+    ],
+    "OpenSlides": [
+        {
+            "name": "help wanted",
+            "color": "d4c5f9",
+        },
+        {
+            "name": "meta",
+            "color": "1d76db",
+        },
+        {
+            "name": "OpenSlides 3",
+            "color": "006b75",
+        },
+        {
+            "name": "OpenSlides 4",
+            "color": "317796",
+        },
+        {
+            "name": "question",
+            "color": "c5def5",
+        },
+    ],
+    "openslides-backend": [
+        {
+            "name": "migration",
+            "color": "a046d0",
+        },
+        {
+            "name": "performance",
+            "color": "006b75",
+        },
+        {
+            "name": "tests",
```

Comment:
Proposal: Move `tests` to general.

Reply:
Yes, I thought about this, too, but this would clash with the client-specific label `tests` which has another meaning. @m-schieder @MSoeb would it be okay for you to remove the `tests` label in the client? Or rename it?

Reply:
Done.